src/sing/sequencerStateMachine/states/editNotesToolIdleState.ts

```diff
@@ -25,67 +27,84 @@ export class EditNotesToolIdleState
+          executeNotesSelectionProcess(context, input.mouseEvent, input.note);
+          setNextState("moveNote", {
+            cursorPosAtStart: input.cursorPos,
+            targetTrackId: selectedTrackId,
+            targetNoteIds: context.selectedNoteIds.value,
+            mouseDownNoteId: input.note.id,
+            returnStateId: this.id,
+          });
+        } else if (input.targetArea === "NoteLeftEdge") {
+          executeNotesSelectionProcess(context, input.mouseEvent, input.note);
+          setNextState("resizeNoteLeft", {
             cursorPosAtStart: input.cursorPos,
+            targetTrackId: selectedTrackId,
+            targetNoteIds: context.selectedNoteIds.value,
+            mouseDownNoteId: input.note.id,
             returnStateId: this.id,
           });
-        } else if (isOnCommandOrCtrlKeyDown(input.mouseEvent)) {
-          void context.store.actions.DESELECT_ALL_NOTES();
-        } else {
-          void context.store.actions.DESELECT_ALL_NOTES();
-          setNextState("addNote", {
+        } else if (input.targetArea === "NoteRightEdge") {
+          executeNotesSelectionProcess(context, input.mouseEvent, input.note);
+          setNextState("resizeNoteRight", {
             cursorPosAtStart: input.cursorPos,
             targetTrackId: selectedTrackId,
+            targetNoteIds: context.selectedNoteIds.value,
+            mouseDownNoteId: input.note.id,
             returnStateId: this.id,
           });
         }
-      } else if (input.targetArea === "Note") {
-        executeNotesSelectionProcess(context, input.mouseEvent, input.note);
-        setNextState("moveNote", {
-          cursorPosAtStart: input.cursorPos,
-          targetTrackId: selectedTrackId,
-          targetNoteIds: context.selectedNoteIds.value,
-          mouseDownNoteId: input.note.id,
-          returnStateId: this.id,
-        });
-      } else if (input.targetArea === "NoteLeftEdge") {
-        executeNotesSelectionProcess(context, input.mouseEvent, input.note);
-        setNextState("resizeNoteLeft", {
-          cursorPosAtStart: input.cursorPos,
-          targetTrackId: selectedTrackId,
-          targetNoteIds: context.selectedNoteIds.value,
-          mouseDownNoteId: input.note.id,
-          returnStateId: this.id,
-        });
-      } else if (input.targetArea === "NoteRightEdge") {
-        executeNotesSelectionProcess(context, input.mouseEvent, input.note);
-        setNextState("resizeNoteRight", {
-          cursorPosAtStart: input.cursorPos,
-          targetTrackId: selectedTrackId,
-          targetNoteIds: context.selectedNoteIds.value,
-          mouseDownNoteId: input.note.id,
-          returnStateId: this.id,
-        });
       }
     }
   }
 
-  onExit() {}
+  onExit(context: Context) {
+    context.cursorState.value = "UNSET";
+  }
+
+  private updateCursorState(context: Context, isCommandOrCtrlKeyDown: boolean) {
+    if (isCommandOrCtrlKeyDown) {
+      context.cursorState.value = "UNSET";
```

Comment:
ここもしかしたら`CROSSHAIR`かも･･･？（自信少しなし）
多分そうだと思うので変えさせていただきます！間違ってたらすみません！！
```suggestion
      context.cursorState.value = "CROSSHAIR";
```